recipes/mkl_random/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
+
+if [ `uname` == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.10
```

Comment:
Why is this being set? Also why is it set to 10.10?

Reply:
Thanks for the feedback.  Setting the `MACOSX_DEPLOYMENT_TARGET` [instructs](https://stackoverflow.com/questions/6591988/best-practice-for-osx-10-x-backwards-compatability) XCode to generate binaries usable with the set version of OSX and possibly with earlier versions, although it is not guaranteed. Compatibility with earlier versions depends on what features are being utilized by the package. 

The `10.10` value was chosen to align with [Intel(R) Distribution for Python*](https://software.intel.com/en-us/distribution-for-python).

Reply:
Right. So we set this to [`10.9` globally]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/2f8f51a2dd38f8d5ffcef7d4de27e3299cdb9038/recipe/run_conda_forge_build_setup_osx#L9 ), which is also what Anaconda Inc. is using (if I'm not mistaken). Does that work for you or do you need 10.10 for this for some reason?

Reply:
Cool. In such a case, I am removing this custom setting altogether. Thanks for clarifying.